config/config.go

```diff
@@ -41,8 +41,8 @@ func Init() {
 
 	// set mongo server to default if it is not set as a system variable
 	if MONGOSERVER == "" {
-		//MONGOSERVER = "mongodb://spankie:506dad@ds163738.mlab.com:63738/btshopng"
-		MONGOSERVER = "127.0.0.1:27017"
+		MONGOSERVER = "mongodb://spankie:506dad@ds163738.mlab.com:63738/btshopng"
```

Comment:
I've done a lot to install mongo db on my system. even tried using another distro(Kali linux). The binaries it needs to run are hard to find, I really do not have the time now for it. I've really tried alot to make it work. I would just make it a system variable...I really do not mind logging in to mlab... Thanks